zenodo/modules/records/indexer.py

```diff
@@ -30,13 +30,11 @@
 def indexer_receiver(sender, json=None, record=None, index=None,
                      **dummy_kwargs):
     """Connect to before_record_index signal to transform record for ES."""
+    if not index.startswith('records-'):
+        return
     # Inject timestamp into record.
     json['_created'] = record.created
```

Comment:
I think we need these timestamps for grants and funders, but probably we should add these two lines to [Invenio-Indexer](https://github.com/inveniosoftware/invenio-indexer/blob/master/invenio_indexer/api.py#L297) by default.